### PR TITLE
sams: add initial doc page

### PR DESCRIPTION
**Pull request by unknwon**

### CODE REVIEW

**Review by kalanchan — APPROVED** [review]
READY FOR LAUNCH GOOG

File: docs/sourcegraph-accounts/index.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## FAQs
+
+### Do Sourcegraph Enterprise users use Sourcegraph Accounts?
+
+No, Sourcegraph Enterprise users (both on Cloud and on-prem) do <i>not</i> use Sourcegraph Accounts. They continue using the SSO capability built into the Sourcegraph application.
+
+### Do users need to recreate accounts if they previously had accounts on Sourcegraph.com?
+
+No, all of the existing Sourcegraph.com user accounts have been migrated to Sourcegraph Accounts, users should be able to use the same password (if had set) and social sign-in options (Google, GitHub, and GitLab) to sign in Sourcegraph Accounts and get access to the same account on Sourcegraph.com.
```
Comment (MaedahBatool) [inline docs/sourcegraph-accounts/index.mdx:15]:
```suggestion
No, all of the existing Sourcegraph.com user accounts have been migrated to Sourcegraph Accounts, users should be able to use the same password (if had set) and social sign-in options (Google, GitHub, and GitLab) to sign in to Sourcegraph Accounts and get access to the same account on Sourcegraph.com.
```

**Review by MaedahBatool — CHANGES_REQUESTED** [review]
Leaving some small fixes. Rest looks good.

File: docs/sourcegraph-accounts/index.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Sourcegraph Accounts
+
+Sourcegraph Accounts (https://accounts.sourcegraph.com) is the centralized accounts system for all of the Sourcegraph-operated services, it provides Single Sign-On (SSO) experience for our users.
+
+The system is designed to be and operated with high security standards to protect Personally Identifiable Information (PII), secure third-party credentials storage, abuse mitigation and gated infrastructure access.
```
[the remark this answers is not in the document]
Comment (unknwon) [reply docs/sourcegraph-accounts/index.mdx:5]: Hmm, this changes the meaning.

[the remark this answers is not in the document]
Comment (unknwon) [reply docs/sourcegraph-accounts/index.mdx:5]: Gonna apply without deleting the "and"

**Comment by unknwon** [comment]
Merging as-is since this PR is blocking officially launching SAMS (and thus Cody PLG), happy to address post-merge feedback!

**Pull request merged**